cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/profiles.py

```diff
@@ -108,12 +112,16 @@ def load_complete_profiles(file_path: str) -> List[AzureProfile]:
     profiles = load_incomplete_profiles(file_path)
 
     for profile in profiles:
-        raise_on_missing_fields(profile)
-        raise_on_invalid_storage(profile)
+        validate_profile_configuration(profile)
 
     return profiles
 
 
+def validate_profile_configuration(profile: AzureProfile) -> None:
```

Comment:
Since this function (and the two that is calls) operate entirely on instances of `AzureProfile` they can as well be instance methods of that class.

Reply:
My idea was to keep AzureProfile just a data bag, it only carries public data.
There is a bunch of functions that operate on these fields and they can all be replaced with methods; this wont help and wont hurt.